<?xml version="1.0" ?>
<sdf version="1.7">
  <world name="default">
    <gravity>0 0 -9.8</gravity>
    <magnetic_field>6e-06 2.3e-05 -4.2e-05</magnetic_field>
    <atmosphere type="adiabatic"/>
    <max_contacts>100</max_contacts>
    <physics type="ode">
      <max_step_size>0.001</max_step_size>
      <real_time_factor>1</real_time_factor>
      <real_time_update_rate>1000</real_time_update_rate>
      <constraints>
        <cfm>0.2</cfm>
        <erp>0.2</erp>
        <contact_max_correcting_vel>10s</contact_max_correcting_vel>
        <contact_surface_layer>0.0001</contact_surface_layer>
      </constraints>
    </physics>
    <scene>
      <ambient>0.4 0.4 0.4 1</ambient>
      <background>0.7 0.7 0.7 1</background>
      <shadows>1</shadows>
    </scene>
    <spherical_coordinates>
      <surface_model>EARTH_WGS84</surface_model>
      <latitude_deg>0</latitude_deg>
      <longitude_deg>0</longitude_deg>
      <elevation>0</elevation>
      <heading_deg>0</heading_deg>
    </spherical_coordinates>
    <include>
      <uri>model://ground_plane</uri>
    </include>
    <include>
      <uri>model://sun</uri>
    </include>
    <include>
      <uri>model://Table</uri>
      <name>table</name>
      <pose>0 0 0 0 0 0</pose>
      <static>1</static>
    </include>
    <include>
      <uri>model://cabinet_1076</uri>
      <name>cabinet</name>
      <pose>-1.45 -0.2 0.693 0 0 -3.1416</pose>
      <scale>1 1 1</scale>
      <static>0</static>
    </include>
    <model name="apple">
      <static>0</static>
      <link name="link">
        <inertial>
          <pose>0.00127 -0.003882 0.036366 0 -0 0</pose>
          <mass>0.1</mass>
          <inertia>
            <ixx>6.21738e-05</ixx>
            <ixy>9.21744e-08</ixy>
            <ixz>2.35655e-07</ixz>
            <iyy>6.31632e-05</iyy>
            <iyz>-1.42386e-07</iyz>
            <izz>6.08056e-05</izz>
          </inertia>
        </inertial>
        <collision name="collision">
          <geometry>
            <mesh>
              <scale>0.791121 0.791121 0.791121</scale>
              <uri>model://013_apple/google_16k/textured.obj</uri>
            </mesh>
          </geometry>
          <surface>
            <friction>
              <ode>
                <mu>10</mu>
                <mu2>10</mu2>
              </ode>
              <torsional>
                <coefficient>10</coefficient>
                <use_patch_radius>1</use_patch_radius>
                <patch_radius>0.1</patch_radius>
                <ode/>
              </torsional>
            </friction>
            <contact>
              <ode>
                <max_vel>0.001</max_vel>
                <min_depth>0.001</min_depth>
              </ode>
            </contact>
            <bounce/>
          </surface>
          <max_contacts>10</max_contacts>
        </collision>
        <visual name="visual">
          <geometry>
            <mesh>
              <scale>0.791121 0.791121 0.791121</scale>
              <uri>model://013_apple/google_16k/textured.obj</uri>
            </mesh>
          </geometry>
          <material>
            <script>
              <uri>model://013_apple/apple.material</uri>
              <name>apple</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <pose>-0.204362 -0.332023 0.815299 0.003503 -0.001058 -0.625238</pose>
    </model>
    <model name="banana">
      <link name="link">
        <inertial>
          <pose>-0.000671 0.005194 0.015295 -1.5145 1.356 2.81684</pose>
          <mass>0.13268</mass>
          <inertia>
            <ixx>0.000308766</ixx>
            <ixy>1.03519e-20</ixy>
            <ixz>2.03037e-20</ixz>
            <iyy>0.000281297</iyy>
            <iyz>-1.79403e-20</iyz>
            <izz>4.5599e-05</izz>
          </inertia>
        </inertial>
        <collision name="collision">
          <geometry>
            <mesh>
              <scale>1 1 1</scale>
              <uri>model://011_banana/google_16k/textured.obj</uri>
            </mesh>
          </geometry>
          <surface>
            <friction>
              <ode>
                <mu>10</mu>
                <mu2>10</mu2>
              </ode>
              <torsional>
                <coefficient>10</coefficient>
                <use_patch_radius>1</use_patch_radius>
                <patch_radius>0.1</patch_radius>
                <ode/>
              </torsional>
            </friction>
            <contact>
              <ode>
                <max_vel>0.001</max_vel>
                <min_depth>0.001</min_depth>
              </ode>
            </contact>
            <bounce/>
          </surface>
          <max_contacts>10</max_contacts>
        </collision>
        <visual name="visual">
          <geometry>
            <mesh>
              <scale>1 1 1</scale>
              <uri>model://011_banana/google_16k/textured.obj</uri>
            </mesh>
          </geometry>
          <material>
            <script>
              <uri>model://011_banana/banana.material</uri>
              <name>banana</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <pose>-0.370036 0.452791 0.814685 0.014824 0.009645 -2.73055</pose>
    </model>
    <model name="white_ceramic_plate">
      <link name="link">
        <inertial>       
          <pose>0.0007390488445934092 -0.0006078520007953306 0.02064993745562509 -1.4242651966198445 -1.5606244478097873 -0.46156002759133496</pose> 
          <mass>1.1083496692072212</mass>
          <inertia>
            <ixx>0.01139843853170379</ixx>
            <ixy>3.321984134765741e-19</ixy>
            <ixz>-4.8947696390210735e-18</ixz>
            <iyy>0.005826733187054312</iyy>
            <iyz>-8.425868405282165e-19</iyz>
            <izz>0.005798847817602379</izz>
          </inertia>
        </inertial>
        <collision name="collision">
          <geometry>
            <mesh>
              <scale>1 1 1</scale>
              <uri>model://Threshold_Dinner_Plate_Square_Rim_White_Porcelain/meshes/model.obj</uri>
            </mesh>
          </geometry>
          <surface>
            <friction>
              <ode>
                <mu>10</mu>
                <mu2>10</mu2>
              </ode>
              <torsional>
                <coefficient>10</coefficient>
                <use_patch_radius>1</use_patch_radius>
                <patch_radius>0.1</patch_radius>
                <ode/>
              </torsional>
            </friction>
            <contact>
              <ode>
                <max_vel>0.001</max_vel>
                <min_depth>0.001</min_depth>
              </ode>
            </contact>
            <bounce/>
          </surface>
          <max_contacts>10</max_contacts>
        </collision>
        <visual name="visual">
          <geometry>
            <mesh>
              <scale>1 1 1</scale>
              <uri>model://Threshold_Dinner_Plate_Square_Rim_White_Porcelain/meshes/model.obj</uri>
            </mesh>
          </geometry>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <pose>-0.01908 0.203189 0.8137 0.00096 -0.013757 0.015489</pose>
    </model>
    <model name="white_and_pink_box">
      <link name="link">
        <inertial>
          <pose>1.7e-05 -0.000146 0.0568 0 -0 0</pose>
          <mass>0.1</mass>
          <inertia>
            <ixx>0.000133218</ixx>
            <ixy>-1.82144e-08</ixy>
            <ixz>-8.00998e-07</ixz>
            <iyy>0.000133544</iyy>
            <iyz>2.1946e-06</iyz>
            <izz>5.31772e-05</izz>
          </inertia>
        </inertial>
        <visual name="visual">
          <geometry>
            <mesh>
              <scale>0.8 0.8 0.8</scale>
              <uri>model://FemDophilus/meshes/model.obj</uri>
            </mesh>
          </geometry>
        </visual>
        <collision name="collision_0">
          <geometry>
            <mesh>
              <scale>0.8 0.8 0.8</scale>
              <uri>model://FemDophilus/meshes/model.obj</uri>
            </mesh>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <friction>
              <ode>
                <mu>10</mu>
                <mu2>10</mu2>
              </ode>
              <torsional>
                <coefficient>10</coefficient>
                <use_patch_radius>1</use_patch_radius>
                <patch_radius>0.1</patch_radius>
                <ode/>
              </torsional>
            </friction>
            <contact>
              <ode>
                <max_vel>0.001</max_vel>
                <min_depth>0.001</min_depth>
              </ode>
            </contact>
            <bounce/>
          </surface>
        </collision>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <pose>-0.383764 -0.462874 0.814881 -0.003409 -0.006173 -0.036599</pose>
    </model>
    <model name='brown_box'>
      <link name='link'>
        <inertial>
          <pose>8.5e-05 -3.5e-05 0.036814 0 -0 0</pose>
          <mass>0.1</mass>
          <inertia>
            <ixx>7.27895e-05</ixx>
            <ixy>1.06947e-07</ixy>
            <ixz>8.88626e-09</ixz>
            <iyy>7.30272e-05</iyy>
            <iyz>-1.6672e-07</iyz>
            <izz>5.90056e-05</izz>
          </inertia>
        </inertial>
        <visual name='visual'>
          <geometry>
            <mesh>
              <scale>0.8 0.8 0.8</scale>
              <uri>model://Perricone_MD_Neuropeptide_Firming_Moisturizer/meshes/model.obj</uri>
            </mesh>
          </geometry>
        </visual>
        <collision name='collision_0'>
          <geometry>
            <mesh>
              <scale>0.8 0.8 0.8</scale>
              <uri>model://Perricone_MD_Neuropeptide_Firming_Moisturizer/meshes/model.obj</uri>
            </mesh>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <friction>
              <ode>
                <mu>10</mu>
                <mu2>10</mu2>
              </ode>
              <torsional>
                <coefficient>10</coefficient>
                <use_patch_radius>1</use_patch_radius>
                <patch_radius>0.1</patch_radius>
                <ode/>
              </torsional>
            </friction>
            <contact>
              <ode>
                <max_vel>0.001</max_vel>
                <min_depth>0.001</min_depth>
              </ode>
            </contact>
            <bounce/>
          </surface>
        </collision>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <pose>-0.032555 -0.141017 0.814566 0.005934 -0.011096 0.007923</pose>
    </model>
  </world>
</sdf>